### ISSUE TITLE
High "user data" usage

### ISSUE THREAD

**Issue by UvixCreative** [bubelov/news#218]
### Checklist

- [x] I am using the latest version 
- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] This issue contains only one bug. I will open one issue for every bug report I want to file.

### Steps to reproduce the bug

Not entirely sure. I've used the app normally for nearly a year. I have "Show preview images" enabled. I also cleared the cache using Android settings.

### Actual behavior

Very high "user data" usage in Android settings. I do not use the app for podcasts, but my "user data" value is at about 2.5 GB. I assume this is related to preview images being classified as "user data" instead of "cache," but I haven't looked through any code to verify this.

### Expected behavior

Clearing cache in Android settings removes all preview images and requires them to be re-downloaded.

### Screenshots/Screen recordings

![image](https://github.com/bubelov/news/assets/25066003/7ce0a6c7-b0a5-4cab-8c28-36eb01189e87)

### Logs

N/A

### Device info

 - Android version/Custom ROM version: Android 13, kernel version 5.4.210-qgki-perf-g9e40f186e66d
 - Device model: Asus Zenfone 8

**Comment by bubelov** [comment]
Yeah it was a known issue, but now the app is using a different way of fetching images since the old lib was deprecated.

I've been running the preview build for a few days and Android says the app has 2 MB of core state and 25 MB of cache, so I assume the pics are now treated as cache. I tried cleaning the cache and the pics were gone but the app continued to work, so I assume this issue is gone.
